From b42405197cc3cec13021a943d8d15e576ccd7966 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
Date: Mon, 24 Nov 2008 12:05:29 -0600
Subject: [PATCH] ENGR00098874 MX51: Need DDR clock to be set at 166MHz

Changed the parent clock of ddr to be axi_a_clk instead of emi_slow.

Signed-off-by: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
---
 arch/arm/mach-mx51/clock.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx51/clock.c b/arch/arm/mach-mx51/clock.c
index 86de887..b949ddb 100644
--- a/arch/arm/mach-mx51/clock.c
+++ b/arch/arm/mach-mx51/clock.c
@@ -2336,7 +2336,7 @@ static int _clk_ddr_set_parent(struct clk *clk, struct clk *parent)
 
 static struct clk ddr_clk = {
 	.name = "ddr_clk",
-	.parent = &emi_slow_clk,
+	.parent = &axi_a_clk,
 	.set_parent = _clk_ddr_set_parent,
 	.flags = RATE_PROPAGATES,
 };
@@ -2751,7 +2751,7 @@ int __init mxc_clocks_init(void)
 	clk_enable(&main_bus_clk);
 	clk_enable(&gpt_clk[1]);
 
-	clk_set_parent(&ddr_clk, &emi_slow_clk);
+	clk_set_parent(&ddr_clk, &axi_a_clk);
 
 	/* Set the current working point. */
 	cpu_wp_tbl = get_cpu_wp(&cpu_wp_nr);
-- 
1.5.4.4

